app/helpers/application_helper/toolbar/physical_chassis_center.rb

```diff
@@ -0,0 +1,2 @@
+class ApplicationHelper::Toolbar::PhysicalChassisCenter < ApplicationHelper::Toolbar::Basic
+end
```

Comment:
You need an emty toolbar class? Why is that?

Reply:
I thought that the toolbar class was required. I removed, as we are not adding any action.